firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -423,6 +423,7 @@ private String parseErrorResponseMessage(InputStream inputStream) {
       }
     } catch (IOException ex) {
       // Unable to parse error message.
+      return "Unable to connect to the server, access is forbidden. Http Status code: 403";
```

Comment:
Same "HTTP" capitalization here. Would also call this method `parseForbiddenResponseMessage` if it's 403-specific and add a comment why we're trying to parse it for future reference